test/unit/credit_card_methods_test.rb

```diff
@@ -187,6 +187,16 @@ def test_19_digit_maestro_uk
     assert_equal 'maestro', CreditCard.brand?(number)
   end
 
+  def test_carnet_cards
+    numbers = [
+      '5062280000000000'
```

Comment:
Are there any further dummy numbers you could throw in here for exercising the recognition?

Reply:
Not _yet_; there aren't even test numbers as such. I have two live card numbers, and that's it. I'm open to ideas here, though; I obviously was planning to have more card numbers than I ended up having...

Reply:
Well I'm mostly thinking that it doesn't really matter if they're test cards or not as long as they qualify for the matching criteria. Right?

Reply:
I'll add a few more test cards to make sure we're flexing at least _one_ of the card buckets and at least a couple splotches of the BIN range.